docs/references/express/migrate-from-node-to-express.mdx

```diff
@@ -0,0 +1,104 @@
+---
+title: Clerk Express SDK
+description: Learn how to migrate from Clerk Node SDK to Express SDK.
+---
+
+# Clerk Express SDK
+
+## Migrating from Clerk Node SDK to Clerk Express SDK
+
+<Steps>
+
+### Uninstall @clerk/clerk-sdk-node and install @clerk/express
+
+<CodeBlockTabs type="installer" options={["npm", "yarn", "pnpm"]}>
+```sh filename="terminal"
+npm uninstall @clerk/clerk-sdk-node
+```
+
+```sh filename="terminal"
+yarn remove @clerk/clerk-sdk-node
+```
+
+```sh filename="terminal"
+pnpm remove @clerk/clerk-sdk-node
+```
+
+</CodeBlockTabs>
+
+<CodeBlockTabs type="installer" options={["npm", "yarn", "pnpm"]}>
+```sh filename="terminal"
+npm install @clerk/express
+```
+
+```sh filename="terminal"
+yarn add @clerk/express
+```
+
+```sh filename="terminal"
+pnpm add @clerk/express
+```
+</CodeBlockTabs>
+### Require Authentication
+
+**Using Node SDK**
+
+To enforce strict authentication, use `ClerkExpressRequireAuth()`. This middleware will raise an error when an unauthenticated request is made.
+You can find an example [here](/docs/backend-requests/handling/nodejs#clerk-express-require-auth).
+
+**Using Express SDK**
+
+- **Centralized Middleware:**
+Use `clerkMiddleware` to authenticate all incoming requests without blocking them. This middleware will handle the handshake mechanism.
```

Comment:
If we're gonna reference the handshake mechanism we need to explain what that is or link to a document that does so. Otherwise this is just confusing for users

Reply:
let's remove the reference until we have a dedicated page that explains the handshake.